ui/apps/platform/src/services/DelegatedRegistryConfigService.ts

```diff
@@ -6,6 +6,7 @@ export type DelegatedRegistryConfigEnabledFor = 'NONE' | 'ALL' | 'SPECIFIC';
 export type EnabledSelections = Exclude<DelegatedRegistryConfigEnabledFor, 'NONE'>;
 
 export type DelegatedRegistry = {
+    uuid?: string;
```

Comment:
maybe we can add a comment here explaining why we have this for the FE for context